java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1672,6 +1673,55 @@ public void xadd_and_xtrim(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void xread(BaseClient client) {
+        String key1 = "key1";
+        String key2 = "{" + key1 + "}-1-" + UUID.randomUUID();
+        String field1 = "f1_";
+        String field2 = "f2_";
+        String field3 = "f3_";
+
+        // setup first entries in streams key1 and key2
+        Map<String, String> timestamp_1_1_map = new LinkedHashMap<>();
```

Comment:
timestamp? I think it is `id`

Reply:
note that every entry in a stream has KV pairs like a map

Reply:
I agree, but this is the name of the variable used in node